hols/todo/MultiCluster.md

```diff
@@ -0,0 +1,357 @@
+# Multi-Cluster Deployment
+
+## A. Infrastructure
+
+### Provision OKE Clusters
+
+1. Use the terraform module in infra/multi-cluster:
+
+```bash
+cd infra/multi-cluster
+cp terraform.tfvars.example terraform.tfvars
+```
+
+2. Edit terraform.tfvars and enter appropriate values for:
+- `api_fingerprint`
+- `api_private_key_path`
+- `home_region`
+- `tenancy_id`
+- `user_id`
+- `compartment_id`
+- `ssh_private_key_path`
+- `ssh_public_key_path`
+- `kubernetes_version`
+- `timezone`
+- `istio_version`
+- `istio_mesh_id`
+3. Edit the clusters input variable and set the appropriate regions:
+
+```
+clusters = {
+  c1 = { region = "paris", vcn = "10.1.0.0/16", pods = "10.201.0.0/16", services = "10.101.0.0/16", enabled = true }
+  c2 = { region = "marseille", vcn = "10.2.0.0/16", pods = "10.202.0.0/16", services = "10.102.0.0/16", enabled = true }
+  # c3 = { region = "frankfurt", vcn = "10.3.0.0/16", pods = "10.203.0.0/16", services = "10.103.0.0/16", enabled = true }  
+}
+    
+}
+```
+4. Run terraform init and apply:
+
+```
+terraform init
+terraform apply --auto-approve
+```
+5. RPCs will have been already created. Login to OCI Console and peer the VCNs using the RPCs.
+
+### Configure Multi cluster
+1. ssh to the operator host
+
+2. Deploy Cilium in all clusters:
+```bash
+for c in c1 c2; do
+  kubectx $c
+  helm install cilium cilium/cilium --namespace=kube-system -f $HOME/cilium/cilium-$c.yaml
```

Comment:
Where is`$HOME/cilium/cilium-$c.yaml` coming from ?

Reply:
It'll be created and uploaded to the operator host by terraform